contrib/platform/src/com/sun/jna/platform/win32/WinNT.java

```diff
@@ -2178,6 +2178,11 @@ public EVENTLOGRECORD(Pointer p) {
 	int PROCESS_TERMINATE = 0x00000001;
 
 	/**
+	 * Required for getting process path
+	 */
+	int PROCESS_NAME_NATIVE = 0x00000001;
```

Comment:
Sorry missed this one. The comment above is incorrect, MSDN documents this as "The name should use the native system path format." or something like that, sync this with whatever is there or in the winnt.h header, please.